saskatoon/member/admin.py

```diff
@@ -259,13 +270,35 @@ class PersonAdmin(admin.ModelAdmin):
         'auth_user__email',
     )
 
+    @admin.display(description="AuthUser")
+    def authuser(self, person):
+        try:
+            user = person.auth_user
+        except AuthUser.DoesNotExist:
+            return None
+        base_url = reverse('admin:member_authuser_changelist')
```

Comment:
Note: using `reverse('admin:member_authuser_change', user.id) to get the full url right away would raise circular import issue